client-app/pages/account/checkout-defaults.vue

```diff
@@ -34,26 +34,28 @@
           size="auto"
         >
           <template #placeholder>
-            <VcSelectItem>
-              <VcSelectItemImage src="/static/icons/placeholders/select-payment.svg" class="bg-gray-100/80" />
-              <VcSelectItemText>
-                {{ $t("common.placeholders.select_payment_method") }}
-              </VcSelectItemText>
-            </VcSelectItem>
+            <div class="flex items-center gap-3 p-3 text-sm">
```

Comment:
Why do we moved from component to duplicated `div`'s with same classes? Looks like regression for me.
I think it's better to have component here, may be customizable. If we don't want to have specific for `VcSelect`, then reuse `MenuItem` or something like that.

Reply:
We have new design for selects https://virtocommerce.atlassian.net/browse/VCST-468